### ISSUE TITLE
Arithmetic: subtract doesn't

### ISSUE THREAD

**Issue by davidaturton** [zim-desktop-wiki/zim-desktop-wiki#1116]
There seems to be a problem with the subtraction operation in the Arithmetic plugin:

> 3+2=5
> 3-2=3
> 3+(-2)=1

Zim 0.72.1 on Linux-Solus
Many thanks, David

**Comment by Nick222** [comment]
Confirm
Xubuntu 20.04
Zim 0.73

But when I add space between symbols - all became OK

And - author of Arithmetic plugin - Patricio Paez (not Jaap Karssenberg).

**Comment by pcampr** [comment]
Confirming. The trick with spaces works, thanks!
```
8-1=8
8--1=9
8---1=7
-1=-1

8 - 1 =7
```
Zim 0.74.2, Kubuntu 21.04

**Comment by introt** [comment]
For clarity: the subtraction operator `-` doesn't work if not separated by spaces; this is not the case for other operators, such as `+`

> 3-2=3

**Comment by jtoppine** [comment]
Sorry for the bump, but confirming; still an issue with Zim 0.75.2 / Linux Mint 21.2 (Flatpak, since Flathub has the newer Zim version)

**Comment by introt** [comment]
The `zim.inc.arithmetic` is pretty hard to parse; as a hacky workaround, we could force spaces around `-` before parsing. Are there any edge cases (eg. related to order of operations) besides having too many spaces? (`8 -  1=1`)
